src/components/pages/blogs/blog/components/SimpleTemplate.tsx

```
@@ -3,17 +3,28 @@ import React from "react";
+                    📝
+                </div>
+                <h2 className="text-2xl lg:text-3xl font-bold text-gray-800 leading-tight">
+                    {content.title}
+                </h2>
+            </div>
+
+            <div className="prose prose-lg max-w-none">
+                {content.description?.split("\n").map((paragraph, index) => (
+                    <p className="text-gray-700 leading-relaxed mb-6 text-base lg:text-lg"
+                        key={index}
+                        dangerouslySetInnerHTML={{ __html: paragraph }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)